src/checks/cargo/cargo.ts

```diff
@@ -96,7 +115,14 @@ function getPrimaryOrFirstSpan(spans: DiagnosticSpan[]): DiagnosticSpan | null {
   return primarySpan ?? spans[0]
 }
 
-export function cargoHasBuildFinished(output: CargoMessage[]): boolean {
+export function cargoHasBuildFinishedSuccessfully(output: CargoMessage[]): boolean {
```

Comment:
The name is kinda long, I suggest something like `cargoBuildsSuccessfully` or similar

Reply:
One issue with that name, is that it doesn't suggest a boolean is returned, where functions usually tell that by using `is` and `has`

Reply:
Then `isCargoBuildSuccess` or `isCargoBuiltSuccessfully`

Reply:
Now it just sounds like broken English :)

Reply:
Not if you write `isCargoBuildSuccessfully` - referring to `cargo build`